global.json

```diff
@@ -1,6 +1,8 @@
 {
+  "$schema": "http://json.schemastore.org/global",
   "sdk": {
-    "version": "3.1",
+    "version": "5.0.301",
```

Comment:
Any reason for going to 5.0.301 rather than a specific 3.1 SDK? I'd really prefer to stick to LTS SDK releases if at all possible.

Reply:
That's because the `NodaTime.TzdbCompiler` project multi-targets `netcoreapp3.1` + `net5.0` (introduced in fb880b06a2c41def4f1f9a218703d302e3523492). The `net5.0` target framework requires the .NET 5 SDK and since the `global.json` was misconfigured, the SDK 5 was actually always used.

I have removed the `net5.0` target in order to stay on the .NET Core 3.1 LTS SDK in 54f3bf9e8db5c16a0b3d801de5ece0e1d83e1e2e.

Reply:
Aha... okay. I'll need to have a think about this. I can't remember offhand why I wanted to be able to run the TzdbCompiler under .NET 5 explicitly... maybe we'll want to use 5.0 after all. I'm busy for the rest of today, but I'll hopefully get back to you about this tomorrow.

Reply:
You wrote this in commit fb880b06a2c41def4f1f9a218703d302e3523492:
> .NET 5.0 has more instrumentation for null references in LINQ to XML

I'll let you think about it and I'll adapt the version in `global.json` and the target frameworks to whatever you think is the best for Noda Time.

Reply:
Right, I've had a think - I suspect that what you had before, with global.json using the 5.0 SDK, is probably the best course of action for the moment. I'll update it to 6 when that comes out. Sorry about the inconvenience here!

Reply:
I think that's a good call. 👍